exampleCourse/questions/demo/autograder/python/mustacheProcess/clientFilesQuestion/style.css

```diff
@@ -0,0 +1,3 @@
+table, th, td {
+    border: 1px solid;
+}
```

Comment:
Took me a long time to get back to this. Sorry. For this one, I think the best solution is to eliminate the formatting completely. It was only there because of an example I was adapting and needlessly complicates the example. I'll also remove the folder and the reference to this from info.json.